test/common.js

```diff
@@ -138,7 +136,7 @@ function dropDBs(done) {
         if (!mongos) return done();
 
 
-        var db = start({ uri: mongos, mongos: true });
+        var db = mongoose.connect(mongos, {mongos: true });
```

Comment:
I assume this was trying to connect to the database.
It is not defined elsewhere in the file.